### PR TITLE
1.81.0 JSON release notes

### PR DESCRIPTION
**Pull request by grisumbras**

### CODE REVIEW

**Comment by pdimov** [comment]
Should be after Flyweight (in alphabetical order.)

**Comment by grisumbras** [comment]
Sorry, missed it.

File: feed/history/boost_1_81_0.qbk [hunk 1]
```diff
@@ -85,7 +85,17 @@ Please keep the list of libraries sorted in lexicographical order.
 
 * [phrase library..[@/libs/flyweight/index.html Flyweight]:]
   * Maintenance work.
-  
+
+* [phrase library..[@/libs/json/ JSON]:]
+  * Added `object::stable_erase`.
+  * Added `parse` overload for `std::istream` and  `operator>>` for `value`.
+  * Added rvalue ref-qualified accessors for `value`.
+  * Conversion traits were redisigned.
```
Comment (pdimov) [inline feed/history/boost_1_81_0.qbk:93]: Typo, "redisigned".

Comment (pdimov) [reply feed/history/boost_1_81_0.qbk:93]: Actually never mind, I'll just fix that.

**Comment by grisumbras** [comment]
Thanks

**Pull request merged**